docs/terminal/melbourne.md

```diff
@@ -96,9 +93,10 @@ Once the aircraft is ready for departure and no conflict exists in your airspace
     **FDK**: "Hold at the holding point, FDK"  
 
     **ML TCU**: "FDK, make right turn, report airborne"  
-    **FDK**: "Make right turn, FDK"
+    **FDK**: "Make right turn, FDK"  
+    `AIP GEN 3.4`
 
-## YMAV TWR Offline
```

Comment:
YMAV Offline is here, but not YMAV Online

Reply:
AV TWR coord section?

Reply:
@alexd-332 

For example; We place Inbound coordination examples in ELW & AY ADC. They are identical, but feature twice.

Is this because we want to show each controller what to expect using only their SOP page. So some stuff will get duplicated between pages, or are we implying controllers will need to use multiple SOPs to obtain answers to their questions. Doing this, ML TCU will look at EN TWR Online section when they pop up, and see the stuff immediately relevant to them, and what they can expect, without digging through a bunch of stuff they dont need in a seperate SOP.

For the record, I'm not implying an ENR controller running ELW should have everything for ML ADC in their SOP listing. If you are performing the role of multiple position, you'll need their SOPs. But thats not true in this case.

Reply:
is there anything more that a ML TCU controller needs to know about how they operate when EN TWR is online, that's not contained in the ML TCU / EN ADC coord section?

Reply:
That's fair.

Maybe consider changing the title to XX Class G Procedures? 
Or consider moving the section down below coordination section?